KsApi/mutations/inputs/UpdateBackingInput.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public struct UpdateBackingInput: GraphMutationInput, Encodable {
+  let amount: String?
+  let applePay: ApplePayParams?
+  let id: String
```

Comment:
Can you add a comment here to clarify what this `id` is? Is it the `backing.id`? Also should it be the `graphId` or the regular `id`? Come to think of it, it would be useful to have comments on all the `id`s, though I know we haven't done that for other mutation inputs.

Reply:
Added documentation to the init